markdown/developers/tutorials/unofficial_pipelines.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Tutorial: Reusing nf-core components in non-nf-core pipelines. "
```

Comment:
Might be a bit on the long side?

Reply:
Suggestion on how to make it shorter? I went through 3-4 versions and was never happy...

Reply:
`Using components nf-core in your own pipeline`?

Reply:
`Using nf-core components in your own pipeline` +1
or `Using nf-core components outside nf-core`

Reply:
Ooh I like the last one